images/downloader/Dockerfile

```diff
@@ -12,12 +10,15 @@ RUN apt-get update -qq && apt-get install -y -q \
     pip install --upgrade pip && \
     rm -rf /root/.cache/pip*
 
-COPY pkg/workloads/cortex/downloader/requirements.txt /src/cortex/downloader/requirements.txt
-RUN pip install -r /src/cortex/downloader/requirements.txt && \
+COPY pkg/workloads/cortex/internal/requirements.txt /src/cortex/internal/requirements.txt
+COPY pkg/workloads/cortex/downloader /src/cortex/downloader
```

Comment:
Can this `COPY` be moved after the `RUN pip install --no-cache-dir`(line 16-18)?

Reply:
Yes, it can be moved. Will do so.